<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find sobit_mapping)/config/rviz/sobit_navigation.rviz"/>

    <arg name="map_file"      default="$(find sobit_mapping)/map/example.yaml"/>
    <arg name="location_file" default="$(find sobit_mapping)/map/map_location_example.yaml"/>
    
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

    <group ns = "location_file_viewer">
        <node pkg="sobit_mapping" type="location_file_viewer"  name="location_file_viewer" output="screen">
            <param name="location_file_path" type="string" value="$(arg location_file)"/>
        </node>
    </group>
    <rosparam command="load" file="$(arg location_file)"/>
</launch>
